main.go

```diff
@@ -164,6 +167,8 @@ func main() {
 			"If omitted, the default Go cipher suites will be used. \n"+
 			"Preferred values: "+strings.Join(tlsCipherPreferredValues, ", ")+". \n"+
 			"Insecure values: "+strings.Join(tlsCipherInsecureValues, ", ")+".")
+	flag.IntVar(&bmoConcurrency, "bmo-concurrency", 0,
```

Comment:
Since this is used for everything now, perhaps we could rename it to just `concurrency` (or `controller-concurrency`?) and say it is the "Number of CRs of each type to process simultaneously".

Reply:
done